tools/vpdbootmanager/main.go

```diff
@@ -30,23 +34,31 @@ Global flags:
 
 -vpd-dir - VPD dir to use
 
-`
+`, progname, progname)
+}
 
 func main() {
 	if err := cli(os.Args[1:]); err != nil {
-		fmt.Printf(usage, os.Args[0])
+		fmt.Println(getUsage(os.Args[0]))
 		fmt.Printf("Error: %s\n\n", err)
 		os.Exit(1)
 	}
 }
 
 func cli(args []string) error {
 	if len(args) < 1 {
-		return fmt.Errorf("You need to provide action")
+		return fmt.Errorf("you need to provide action")
 	}
 	switch args[0] {
 	case "add":
 		return add(args[1], args[2:])
+	case "get":
+		varname := ""
+		if len(args) > 1 {
+			varname = args[1]
+		}
+		getter := Getter{}
```

Comment:
I think it should be
getter := NewGetter()

Reply:
indeed it does! I forgot to use it after I added `NewGetter`, good catch

Reply:
done